src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -8690,5 +8690,123 @@ unsafe void M(in int p)
         }
 
         #endregion
+
+        [Fact]
+        public void AddressOfFixedSizeBuffer()
+        {
+            CreateCompilation(@"
+unsafe struct S
+{
+    public fixed int Buf[1];
+}
+
+unsafe class C
+{
+    static S s_f;
```

Comment:
>s_f [](start = 13, length = 3)

Just checking the notation used here... `_f` means "moveable"?

Reply:
Oh, it means "field".

---
In reply to: [203540798](https://github.com/dotnet/roslyn/pull/28527#discussion_r203540798) [](ancestors = 203540798)

Reply:
"field". Maybe I should just write it out... :/